docs/hikey.md

```diff
@@ -57,6 +57,9 @@ before building, see [UART in hikey.mk].
 Just follow the "Get and build the solution" in the [README.md] file. The `make
 flash` step will tell you how you should set the jumpers on the board.
 
+**NOTE:** If you're using the Kirin 620 or Hikey from CircuitCo, please set
```

Comment:
I find the wording a bit weird, because Kirin 620 is the name of the SoC. All HiKey boards, whether from CircuitCo or LeMaker, do have a Kirin 620 (not talking about the HiKey960 of course).